clickhouse_driver/client.py

```diff
@@ -177,8 +177,10 @@ def execute(self, query, params=None, with_column_types=False,
                          Defaults to ``None`` (no additional settings).
         :param types_check: enables type checking of data for INSERT queries.
                             Causes additional overhead. Defaults to ``False``.
-        :param columnar: if specified the result will be returned in
-                         column-oriented form.
+        :param columnar: if specified the result of the SELECT query will be
+                         returned in column-oriented form.
+                         It also allows to INSERT columnar data (`list` of
```

Comment:
You can also insert list of dicts.

Maybe "It also allows to INSERT data in columnar form." will be enough?

Reply:
I have no idea how list of dicts should look for columnar data, so I decided to stick to list of tuples.